ricecooker/utils/kolibripreview.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+import argparse
+import os
+import shutil
+import sys
+
+
+def validate(srcdir):
+    """
+    Check if `srcdir` has an index.html in it.
+    """
+    indexpath = os.path.join(srcdir, 'index.html')
+    if not os.path.exists(indexpath):
+        print('Missing index.html file in', srcdir)
+        return False
+    return True
+
+
+def main(args):
+    """
+    Command line utility for previewing HTML5App content in Kolbri.
```

Comment:
Can we add a note about the fact that this works by replacing an existing HTML5 zip in the local Kolibri installation? I know the destzip arg docstring alludes to it, but I think it would be good to make it clear in the tool documentation.